internal/events/loop_test.go

```diff
@@ -37,8 +37,7 @@ func (r *unsupportedResource) DeepCopyObject() runtime.Object {
 
 var _ = Describe("EventLoop", func() {
 	var ctrl *events.EventLoop
```

Comment:
nit: might be worth throwing these in a var block.

Reply:
thanks. looks more readable and cleaner now!

Reply:
implemented in "Fix FIXMEs about throwing panics"